atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -602,4 +561,9 @@ public enum StartupChecks {
         RUN,
         DO_NOT_RUN
     }
+
+    @VisibleForTesting
+    CassandraAbsentHostTracker getAbsentHostTracker() {
```

Comment:
ugh this is ugly, but the alternative is to have this be passed in via constructor -- we can go down this path, but the constructor of this class is already bloated as it is.

Reply:
alternatively we could abandon this altogether, and just not test absent host tracking (which was the case before!!)